### PR TITLE
Python3 support for robotframework-pykafka

### PR DESCRIPTION
**Pull request by ank15**
Updated message and key format for producer input from string to bytes and consumer output format from bytes to string.
Updated broker version from 2.

### CODE REVIEW

**Review by monksy — DISMISSED** [review]
Looks good to me!

File: robotframework_pykafka/robotframework_pykafka.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def __init__(self):
             self._kafkaBrokerVersion = os.environ['KAFKA_BROKER_VERSION']
         except KeyError as e:
             # Default it if not specified
-            self._kafkaBrokerVersion = "1.0.0"
+            self._kafkaBrokerVersion = "2.3.0"
```
Comment (invadergir) [inline robotframework_pykafka/robotframework_pykafka.py:28]: @ank15  Please update the README to reflect the new default version.

File: robotframework_pykafka/robotframework_pykafka.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def __init__(self):
             self._kafkaBrokerVersion = os.environ['KAFKA_BROKER_VERSION']
```
Comment (invadergir) [inline robotframework_pykafka/robotframework_pykafka.py:25]: Also please update the comment on line 22 with the same version (2.3.0)

**Review by monksy — APPROVED** [review]

**Review by invadergir — APPROVED** [review]
I'll update the comments after the merge...

**Pull request merged**